src/style.css

```diff
@@ -1,3 +1,80 @@
-body {
+* {
```

Comment:
Better not to use * selector, you can set this on those elements you need
![image](https://github.com/mate-academy/layout_moyo-header/assets/46566190/722d12b3-2ef7-4461-9af8-38a68edc8a2b)